test/unit/adapters/http.js

```diff
@@ -670,7 +670,7 @@ describe('supports http with nodejs', function () {
           proxy: {
             host: 'localhost',
             port: 4000,
-            protocol: 'https'
+            protocol: 'https:'
```

Comment:
If it's necessary to support both with and without trailing `:`, a small change is needed in `setProxy` to append it if it's missing (when `options.protocol` is overridden), otherwise it fails with an error in `follow-redirects`.